packages/components/time-picker/src/time-picker-com/basic-time-spinner.vue

```diff
@@ -212,7 +212,14 @@ const adjustSpinner = (type: TimeUnit, value: number) => {
 
 const typeItemHeight = (type: TimeUnit): number => {
   const scrollbar = unref(listRefsMap[type])
-  return scrollbar?.$el.querySelector('li').offsetHeight || 0
+  const listItem = scrollbar?.$el.querySelector('li')
+  if (listItem) {
+    const computedStyle = window.getComputedStyle(listItem)
+    const height = Number.parseFloat(computedStyle.getPropertyValue('height'))
+    return height || 0
```

Comment:
```suggestion
    return Number.parseFloat(getStyle(listItem, 'height')) || 0
```

https://github.com/element-plus/element-plus/blob/798d9d873939815e750786280c4993d25b05bf59/packages/utils/dom/style.ts#L29-L32

Reply:
have been updated